nextflow.config

```
@@ -99,103 +100,109 @@ try {
-// try {
-//   includeConfig "${params.custom_config_base}/pipeline/crisprseq.config"
-// } catch (Exception e) {
-//   System.err.println("WARNING: Could not load nf-core/config/crisprseq profiles: ${params.custom_config_base}/pipeline/crisprseq.config")
-// }
+try {
+    includeConfig "${params.custom_config_base}/pipeline/crisprseq.config"
+} catch (Exception e) {
+    System.err.println("WARNING: Could not load nf-core/config/crisprseq profiles: ${params.custom_config_base}/pipeline/crisprseq.config")
+}
 profiles {
     debug {
-        dumpHashes             = true
-        process.beforeScript   = 'echo $HOSTNAME'
-        cleanup                = false
+        dumpHashes              = true
+        process.beforeScript    = 'echo $HOSTNAME'
+        cleanup                 = false
         nextflow.enable.configProcessNamesValidation = true
     }
     conda {
-        conda.enabled          = true
-        docker.enabled         = false
-        singularity.enabled    = false
-        podman.enabled         = false
-        shifter.enabled        = false
-        charliecloud.enabled   = false
-        channels               = ['conda-forge', 'bioconda', 'defaults']
-        apptainer.enabled      = false
+        conda.enabled           = true
+        docker.enabled          = false
+        singularity.enabled     = false
+        podman.enabled          = false
+        shifter.enabled         = false
+        charliecloud.enabled    = false
+        conda.channels          = ['conda-forge', 'bioconda', 'defaults']
+        apptainer.enabled       = false
     }
     mamba {
-        conda.enabled          = true
-        conda.useMamba         = true
-        docker.enabled         = false
-        singularity.enabled    = false
-        podman.enabled         = false
-        shifter.enabled        = false
-        charliecloud.enabled   = false
-        apptainer.enabled      = false
+        conda.enabled           = true
+        conda.useMamba          = true
+        docker.enabled          = false
+        singularity.enabled     = false
+        podman.enabled          = false
+        shifter.enabled         = false
+        charliecloud.enabled    = false
+        apptainer.enabled       = false
     }
     docker {
-        docker.enabled           = true
-        conda.enabled            = false
-        singularity.enabled      = false
-        podman.enabled           = false
-        shifter.enabled          = false
-        charliecloud.enabled     = false
-        apptainer.enabled        = false
-        process.containerOptions = '-u $(id -u):$(id -g)'
+        docker.enabled          = true
+        conda.enabled           = false
+        singularity.enabled     = false
+        podman.enabled          = false
+        shifter.enabled         = false
+        charliecloud.enabled    = false
+        apptainer.enabled       = false
+        docker.runOptions       = '-u $(id -u):$(id -g)'
```

We changed this to `process.containerOptions` to allow overwriting it on some modules which were failing with these docker options, we should leave the containerOptions
```suggestion
        process.containerOptions       = '-u $(id -u):$(id -g)'
```